qcodes/instrument_drivers/rigol/DSG815.py

```diff
@@ -0,0 +1,51 @@
+from qcodes import VisaInstrument
+from qcodes.instrument.parameter import Parameter
+from qcodes.utils.validators import Numbers, Bool, Enum
+
+class DSG815(VisaInstrument):
+    def __init__(self, name, address, **kwargs):
+        super().__init__(name=name, address=address, terminator='\n', **kwargs)
+        self.reset()
+
+        self.unit = Parameter('unit',
+                              parent=self,
+                              vals=Enum('DBM', 'DBMV', 'DBUV', 'V', 'W'),
+                              get_cmd=':UNIT:POWer?',
+                              set_cmd=':UNIT:POWer {}',
+                              label='RF output signal unit',
+                              docstring='Unit for the RF output level command to use')
+
+        self.power = Parameter('power',
+                               parent=self,
+                               vals=Numbers(-110, 20),
+                               get_cmd=':LEVel?',
+                               set_cmd=':LEVel {}',
+                               unit='dBm',
+                               label='RF output power')
+
+        self.amplitude = Parameter('amplitude',
+                               parent=self,
+                               vals=Numbers(1e-6, 3.162),
+                               get_cmd=':LEVel?',
+                               set_cmd=':LEVel {}V',
+                               unit='V',
+                               label='RF output Vp')
```

Comment:
Power and amplitude parameters are set only for dBm and V, but you allow also to use 'DBMV', 'DBUV', 'W' for units. It might be dangerous.

Reply:
Otherwise, all seems to be fine for simple instrument use.

Reply:
Yeah that's a good point actually. Not an easy one to tackle at that. Not sure what the best option is here

Reply:
I've addressed this now, the `unit` parameter is now `display_unit` to reflect the fact that it only changes what's shown on the instrument front panel.

I've also corrected the use of the amplitude settings, and added a distinct parameter for setting the RMS vs. peak amplitude. Ready for re-review.